src/dxgi/dxgi_adapter.cpp

```diff
@@ -285,7 +285,8 @@ namespace dxvk {
       ? std::string(deviceProp.deviceName)
       : options->customDeviceDesc;
 
-    if (options->customVendorId < 0) {
+    if (options->customVendorId < 0
+    || (deviceProp.vendorID == uint16_t(DxvkGpuVendor::Intel) && options->hasXess)) {
```

Comment:
This check seems redundant unless the user *explicitly* overrides the vendor ID? I don't think we should try to be smart and work around broken user configs, if anything that makes it harder to test things on the fly.

Reply:
Ah, I didn't realize it defaulted to -1, that works perfect then! I'll remove & re-upload.